provider.go

```diff
@@ -0,0 +1,88 @@
+package egcl
+
+import (
+	"bytes"
+	"crypto/rsa"
+	"encoding/json"
+	datahub "github.com/mimiro-io/datahub-client-sdk-go"
+	egdm "github.com/mimiro-io/entity-graph-data-model"
+)
+
+type RemoteDataProvider struct {
+	client *datahub.Client
+}
+
+func NewRemoteDatahubQueryHelper(endpoint string) (*RemoteDataProvider, error) {
```

Comment:
Constructor name and return type name inconsistent

Reply:
Agreed will fix.